rllib/utils/serialization.py

```diff
@@ -325,3 +326,58 @@ def check_if_args_kwargs_serializable(args: Sequence[Any], kwargs: Dict[str, Any
                 f"Found non-serializable keyword argument: {k} = {v}.\n"
                 f"Original serialization error: {e}"
             )
+
+
+@DeveloperAPI
+def serialize_type(type_: Union[Type, str]) -> str:
+    """Converts a type into its full classpath ([module file] + "." + [class name]).
+    Args:
+        type_: The type to convert.
+    Returns:
+        The full classpath of the given type, e.g. "ray.rllib.algorithms.ppo.PPOConfig".
+    """
+    # Already serialized.
+    if isinstance(type_, str):
+        return type_
+
+    return type_.__module__ + "." + type_.__qualname__
```

Comment:
so this is ok for now, but normally we would use a registry and registered name for this.
the reason being, if you moved your source file, or have refactored the class name, etc, you can easily update the registry to make the old checkpoints work.

Reply:
I thought about this as well. Let me add a todo and lets add that.

Reply:
done